tool/gravity/cli/system.go

```diff
@@ -280,6 +282,33 @@ func reinstallOneshotService(env *localenv.LocalEnvironment, serviceName string,
 	return trace.Wrap(err)
 }
 
+// reinstallService stops and reinstalls the service specified by serviceName
+// as a simple service.
+func reinstallService(env *localenv.LocalEnvironment, serviceName string, cmd []string) error {
```

Comment:
I'd really like that we moved general purpose functions to specific packages - i.e. `lib/system/service` if there's no such API yet.